perun-base/src/test/resources/test-schema.sql

```diff
@@ -567,9 +567,9 @@ create table application_data (
 create table application_mails (
 	id integer not null,
 	form_id integer not null,       --identifier of form (application_form.id)
-	app_type varchar(30) not null,  --application type (initial/extension)
-	mail_type varchar(30) not null, --type of mail (user/administrator)
-	send varchar(1) not null,       --sent (Y/N)
+	app_type longvarchar not null,  --application type (initial/extension)
+	mail_type longvarchar not null, --type of mail (user/administrator)
+	send longvarchar not null,       --sent (Y/N)
```

Comment:
Should be probably char(1) as in postgres.

Reply:
fixed